docs/api/reference.md

```diff
@@ -0,0 +1,751 @@
+---
+id: reference
```

Comment:
To stick to the installation, shouldn't we rename it: gemwallet-api-reference ?

Reply:
Done